packages/cfd/src/Containers/mt5-account-unavailable-modal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Button, Modal } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import { isDesktop } from '@deriv/shared';
```

Comment:
Not sure if it applies to cfd package but it's better to use `is_desktop` from ui store instead